pkgs/development/python-modules/django-widget-tweaks/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage, fetchFromGitHub, stdenv, django }:
+
+buildPythonPackage rec {
+  pname = "django-widget-tweaks";
+  version = "1.4.8";
+
+  src = fetchFromGitHub { # package from Pypi missing runtests.py
+    owner = "jazzband";
+    repo = pname;
+    rev = version;
+    sha256 = "00w1ja56dc7cyw7a3mph69ax6mkch1lsh4p98ijdhzfpjdy36rbg";
+  };
+
+  checkPhase = "python runtests.py";
```

Comment:
`${python.interpreter}` instead of `python` because `python` does not always exist (e.g. in case of pypy)

Reply:
Fixed, thank you! Verified that tests work in pypy (`pypy3Packages.django-widget-tweaks`).